### ISSUE TITLE
Package phpoffice/phpexcel is abandoned...

### ISSUE THREAD

**Issue by AmirHome** [LaravelDaily/quickadmin#133]
Package phpoffice/phpexcel is **abandoned**, you should avoid using it. Use phpoffice/phpspreadsheet instead.

**Comment by igorsantos07** [comment]
That warning actually got me wondering if it's even needed. There's no clear sign on the docs why that dependency is required - could very well become something optional.

**Comment by PovilasKorop** [comment]
@igorsantos07 yes, people abandon working for free in favor of feeding their families. Sad, I know.

Also, read this please: https://fman.io/blog/dear-comrade/
